nipype/algorithms/tests/test_tsnr.py

```diff
@@ -1,19 +1,19 @@
 # emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
 # vi: set ft=python sts=4 ts=4 sw=4 et:
 
-from ...testing import (assert_equal, assert_almost_equal, assert_in, utils)
+from ...testing import (assert_equal, assert_almost_equal, utils)
 from ..confounds import TSNR
 from .. import misc
 
-import unittest
+import pytest
 import mock
 import nibabel as nb
 import numpy as np
 import os
-import tempfile
-import shutil
 
-class TestTSNR(unittest.TestCase):
+#NOTE_dj: haven't removed mock (have to understand better)
```

Comment:
sure, I'll give a second try. Since I've never used any mocking or patching, I had problem with understanding how important the differences between the libraries might be for the project. 
I will open a new PR for it, so it will be easier to review.